hypex/reporters/aa.py

```diff
@@ -53,31 +57,29 @@ def _convert_struct_dict_to_dataset(data: Dict) -> Dataset:
         result = []
         for feature, groups in data.items():
```

Comment:
why in typing u use `data: Dict[str, bool]`, but in cicle variable `groups` may have items?